big_test.go

```diff
@@ -247,6 +247,23 @@ func TestBig_Scan(t *testing.T) {
 	// TODO(eric): write this test
 }
 
+func TestBig_ScanInvalid(t *testing.T) {
+	for i, test := range [...]struct {
```

Comment:
Nitpick: I don’t see the point in “...”

Reply:
I didn't either, but all of the nearby tests followed the pattern too :)

Reply:
Yeah, I used to use it for some reason. 😅